public/v4/apps/ferretdb.yml

```diff
@@ -0,0 +1,67 @@
+captainVersion: 4
+services:
+    $$cap_appname-db:
+        image: ghcr.io/ferretdb/postgres-documentdb:$$cap_postgres_version
+        restart: on-failure
+        environment:
+            POSTGRES_USER: $$cap_db_user
+            POSTGRES_PASSWORD: $$cap_db_pass
+            POSTGRES_DB: $$cap_db_name
+        volumes:
+            - $$cap_appname-db-data:/var/lib/postgresql/data
+        caproverExtra:
+            notExposeAsWebApp: 'true'
+
+    $$cap_appname:
+        image: ghcr.io/ferretdb/ferretdb:$$cap_ferretdb_version
+        restart: on-failure
+        ports:
+            - 27017:27017
```

Comment:
Do not expose ports by default. The end user can opt into exposing ports if they need to connect to db externally.